backend/src/main/java/com/example/backend/service/writing/WritingService.java

```diff
@@ -78,7 +84,13 @@ public WritingResponse getAndRecommendById(int id) {
         List<String> users = new ArrayList<>();
         List<MemberLanguage> memberLanguages = memberLanguageRepository.get10ForWriting(languageId, curMemberId);
         for (MemberLanguage m : memberLanguages) {
-            users.add(memberRepository.getOne(m.getMemberId()).getUsername());
+            Member member = memberRepository.getOne(m.getMemberId());
+            users.add(member.getUsername());
+            Notification notification = new Notification();
+            notification.setMemberId(member.getId());
+            notification.setNotificationType(NotificationType.WRITING_EVALUATE);
+            notification.setText("You have a new writing to evaluate!");
+            notificationService.save(notification);
         }
```

Comment:
This part seems wrong, since not all these 10 users are selected as evaluators. They are just recommendations. You should write this code under _processWritingAnswer_ function. @selim-karaduman can correct me if I am wrong.